napari/components/viewer_model.py

```diff
@@ -1137,18 +1147,19 @@ def open(
             )
             plugin = 'napari'
 
-        paths: List[str | Path | List[str | Path]] = (
+        paths_: List[PathLike] = (
```

Comment:
```suggestion
        paths_: list[PathLike] = (
```

Reply:
If we update this List there are a bunch of other we should update everywhere, I would be incline to either have this file all uppercase or all lowercase, but not mix.

Any preference ?